### ISSUE TITLE
Show scheduled events check box

### ISSUE THREAD

**Issue by nobless** [nuxsmin/sysMonDash#31]
In config menu Show scheduled events checkbox not working, even when unticked it shows sheduled events in main screen.

**Comment by nuxsmin** [comment]
Right, this option enables/disables from showing the scheduled events in the list (among other events), but the scheduled downtimes table is always shown on the bottom (if there are such events).

**Comment by nobless** [comment]
okay i understood, but even if i check this option or not i don't see any difference in main dashboard even i have sheduled events
